src/index.ts

```diff
@@ -1,5 +1,26 @@
-function helloWorld(): void {
-    console.log('Hello world');
+function insertFezz(strings: string[]): string[] {
+    let index = strings.findIndex(word => word[0] === "B")
+    if (index === -1) {
+        strings.push("Fezz")
+        return strings
+    }
+    strings.splice(index, 0, "Fezz")
+    return strings
+    
 }
 
-helloWorld();
\ No newline at end of file
+function fizzBuzz(limit: number): void {
+    for (let i = 1; i < limit + 1; i++) {
```

Comment:
When writing typescript it's preferrable to use a more functional approach. Therefore try to not use any for loops and not to mutate any variables (so no `.push` on an array). Instead try to use functions like `.map` and create new variables. The [lodash](https://lodash.com/docs/4.17.15) library has many very useful functions you can use. 

Have a think and see if you can rewrite this with a for loop and without mutating anything. Let me know if you're not sure though and I'm happy to give some more specific suggestions 🙂